setup.py

```diff
@@ -13,8 +13,10 @@
     long_description=open('README.rst').read(),
     url='https://github.com/mozilla/FoxPuppet',
     license='MPL2',
-    packages=['foxpuppet'],
+    packages=[
+        'foxpuppet',
+        'foxpuppet.browser'],
```

Comment:
You can discover modules, but as I've seen issues then trying to exclude test modules, etc.